peerconnection.go

```diff
@@ -592,8 +592,11 @@ func (pc *PeerConnection) CreateAnswer(options *AnswerOptions) (SessionDescripti
 
 // 4.4.1.6 Set the SessionDescription
 func (pc *PeerConnection) setDescription(sd *SessionDescription, op stateChangeOp) error {
-	if pc.isClosed.get() {
+	switch {
+	case pc.isClosed.get():
 		return &rtcerr.InvalidStateError{Err: ErrConnectionClosed}
+	case sd.Type.String() == ErrUnknownType.Error():
```

Comment:
```suggestion
	case newSDPType(sd.Type.String()) == SDPType(Unknown):
```
might be little bit better at semantics